kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/containExactly.kt

```diff
@@ -215,3 +219,12 @@ internal fun<T> StringBuilder.appendPossibleMatches(missing: Collection<T>, expe
       append("\nPrinted first ${AssertionsConfig.maxSimilarityPrintSize.value} similarities out of ${possibleMatches.size}, (set the 'kotest.assertions.similarity.print.size' JVM property to see full output for similarity)\n")
    }
 }
+
+internal fun<T> StringBuilder.appendSubmatches(actual: Collection<T>, expected: Collection<T>) {
```

Comment:
Could this extension function be private? It'd be nice to avoid leaking a utility function specifically for `containsExactly` into other contexts.

Reply:
> Could this extension function be private? It'd be nice to avoid leaking a utility function specifically for `containsExactly` into other contexts.

makes sense. done. TY for calling it out.